README.md

```diff
@@ -95,6 +100,11 @@ For the Java assignments:
 - Maven 3.8.2 or above ([download](http://maven.apache.org/download.cgi))
   - Make sure that Maven uses the correct Java runtime by running `mvn -version`.
 
+For the Python assignments:
+
+- Python 3.9 or above ([download](https://www.python.org/downloads/))
```

Comment:
I do have Python 3 installed - although `python --version` give me 2.7.16, `python3 --version` gives me 3.9.7. I do not have a `pip` command, but I do have `pip3`. I think this is pretty common on macOS, as I believe macOS ships a (very old) Python 2 by default. The instructions tell you to use `pip` and `python`, shouldn't that be `pip3` and `python3` then? Not sure if Python 3.9 installs a `pip3` and a `python3` executable on all platforms...

Reply:
Yep, that's indeed the case on Linux. MacOS doesn't have this issue (I think). I need to mention that in the instructions.

Reply:
I'm in luck today, python3 and pip3 exist as a command on Windows as well. I'm going to replace the statements with that.

Reply:
Fixed with 82bc8405857e24a6487c660233d494ee7cbeb34e and 10694a9eddab59bf7fc81665ba9b79430aba1796